plots/_helpers.py

```diff
@@ -3,14 +3,15 @@
 import pypsa
 import logging
 import yaml
+import pandas as pd
 
 # get the base working directory
 BASE_PATH = os.path.abspath(os.path.join(__file__ ,"../.."))
 # relative path to folder where to store plots
 PATH_PLOTS = "plots/results/"
 
 # Co2L limits
-CO2L_LIMITS = {"2020": "0.725",
+CO2L_LIMITS = {"2020": "0.75",
```

Comment:
I think we can stick to 0.725, I don't know why I tried it before for 0.75. 0.725 creates better results
```suggestion
CO2L_LIMITS = {"2020": "0.725",
```